packages/babel-plugin/src/visitors/stylex-create/parse-stylex-create-arg.js

```diff
@@ -263,3 +247,20 @@ function evaluateObjKey(
     value: String(key),
   };
 }
+
+function validateDynamicStyleParams(
```

Comment:
Why not simplify this function to validate that all params as `t.Identifier` instead?

For example, this pattern will silently ignore array destructuring. We don't really support *anything* other than simple named arguments, so let's just check for that.

Reply:
shouldn't we differentiate between different cases and specify the error message with with what's not allowed?

otherwise it would still be an improvement even if the error specified that `only named parameters are allowed` -- i can simplify it to this as well

Reply:
I think a single simple error message is more helpful, but perhaps add more details. So a single error which says:
`"Only named parameters are allowed. No destructuring. No default values."`

What do you think?

Reply:
yeah, I think it makes sense and I agree. describing one allowed pattern is as effective as trying to describe multiple forbidden patterns, and has less boiler plate. I will restructure the PR.